<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserHelperDialog</class>
 <widget class="QWidget" name="UserHelperDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="dialogGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget{background:white;}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="titleBarWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <layout class="QGridLayout" name="titleBarWidgetGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="titleBarLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="dialogTitleLabel">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(167, 166, 170);</string>
             </property>
             <property name="text">
              <string>Password Manager</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="titleBarLayoutHSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="minimizeButton">
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>最小化</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#e2e2e2;}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../qrc/custom/custom.qrc">
               <normaloff>:/custom/icons/minimize.png</normaloff>:/custom/icons/minimize.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="closeButton">
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>关闭</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#fb7373;}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../qrc/custom/custom.qrc">
               <normaloff>:/custom/icons/close.png</normaloff>:/custom/icons/close.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>106</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="changeUsernameWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#changeUsernameWidget{background:white;border-radius:5px;}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLineEdit" name="username">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>黑体</family>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>输入新用户名</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="changePasswordWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#groupOverviewWidget{background:white;border-radius:5px;}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLineEdit" name="oldPassword">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>黑体</family>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>输入旧密码</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="newPassword">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>黑体</family>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>输入新密码</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="deleteVerifyWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#changeUsernameWidget{background:white;border-radius:5px;}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLineEdit" name="deleteVerify">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>黑体</family>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>输入密码以确认注销用户</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>106</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="bottomWidget" native="true">
        <property name="font">
         <font>
          <family>Microsoft Yi Baiti</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#bottomWidget{background:white;}</string>
        </property>
        <layout class="QGridLayout" name="bottomWidgetGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="bottomLayout">
           <item>
            <spacer name="bottomLayoutSpacerL">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="confirmLayout">
             <item>
              <layout class="QHBoxLayout" name="confirmButtonLayout">
               <item>
                <widget class="QPushButton" name="confirm">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Microsoft YaHei UI</family>
                   <pointsize>10</pointsize>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{color:black;background:#e9e9e9;border:0px solid rgb(167, 166, 170);border-radius:5px;}
QPushButton:hover{background:#d2d2d2;}
QPushButton:press{background:#d2d2d2;}</string>
                 </property>
                 <property name="text">
                  <string>确定</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="cancel">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Microsoft YaHei UI</family>
                   <pointsize>10</pointsize>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{color:black;background:#e9e9e9;border:0px solid rgb(167, 166, 170);border-radius:5px;}
QPushButton:hover{background:#d2d2d2;}
QPushButton:press{background:#d2d2d2;}</string>
                 </property>
                 <property name="text">
                  <string>取消</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="bottomLayoutSpacerR">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qrc/custom/custom.qrc"/>
 </resources>
 <connections/>
</ui>
